crates/uv/tests/venv.rs

```diff
@@ -625,3 +630,109 @@ fn path_with_trailing_space_gives_proper_error() {
     "###
     );
 }
+
+#[test]
+fn create_venv_from_venv() {
+    let context = VenvTestContext::new(&["3.11", "3.12"]);
+
+    // Create a virtual environment at `.venv`.
+    uv_snapshot!(context.filters(), context.venv_command()
+        .arg(context.venv.as_os_str())
+        .arg("--python")
+        .arg("3.12"), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Using Python 3.12.[X] interpreter at: [PATH]
+    Creating virtualenv at: .venv
+    Activate with: source .venv/bin/activate
+    "###
+    );
+
+    context.venv.assert(predicates::path::is_dir());
+
+    // Activate the environment, then create a new virtual environment
+    // We should use the Python interpreter from the active environment
+    uv_snapshot!(context.filters(), context.venv_command()
+        .arg(context.temp_dir.child("bar").as_os_str())
+        .env("VIRTUAL_ENV", context.venv.as_os_str()), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Using Python 3.11.[X] interpreter at: [PATH]
```

Comment:
All these test outcomes are "wrong" given my understanding of the desired behavior.